src/address_format.rs

```diff
@@ -105,10 +105,13 @@ impl<'a> TryFrom<&'a str> for Ss58AddressFormatRegistry {
 	type Error = ParseError;
 
 	fn try_from(x: &'a str) -> Result<Ss58AddressFormatRegistry, Self::Error> {
+		let x = x.to_ascii_lowercase();
+
 		ALL_SS58_ADDRESS_FORMAT_NAMES
-			.binary_search(&x)
+			.into_iter()
+			.position(|n| n.to_ascii_lowercase() == x)
```

Comment:
Could we not directly store them in lowercase?

Reply:
I don't have any idea about this.

Maybe some networks love to keep their name in their style.

Here I just make sure I can find them in a deterministic way.

Reply:
This is internally only. So you should be able to make this lowercase

Reply:
I think this is not private.

`Ss58AddressFormat::all_names() -> &ALL_SS58_ADDRESS_FORMAT_NAMES`